composer.json

```diff
@@ -34,6 +34,8 @@
         "codewithkyrian/chromadb-php": "^0.3.0",
         "doctrine/orm": "^2.20.0",
         "elasticsearch/elasticsearch": "^8.15",
+        "guzzlehttp/guzzle": "^7.4.5",
+        "guzzlehttp/psr7": "^2.7",
```

Comment:
To be able to write test with a concrete HTTP client (e.g. to set a timeout option)